docs/program/run.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Run SDK Code"
+linkTitle: "Run Code"
+weight: 50
+type: "docs"
+description: "Execute the logic you've written to control your robot or fleet."
+images: ["/services/img/icons/sdk.svg"]
+tags: ["client", "sdk", "application"]
+---
+
+After saving your boilerplate code sample and adding control logic with Viam's API methods, run your program to control your Viam-connected robot.
+Do this by running a command to execute the program in the terminal of a machine with the appropriate programming language and Viam SDK installed.
+
+For example:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+python3 ~/myCode/myViamFile.py
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+go run ~/myCode/myViamFile.py
+```
+
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+For an example, see [this execution demo.](https://github.com/viamrobotics/viam-typescript-sdk/tree/main/examples/vanilla)
+
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+For information on running C++ code see [the instructions on GitHub](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+If you are connecting to your robot through an external signaling address, as is the case in the boilerplate code samples generated by the Viam app, you can perform this step on any computer.
+As long as this computer is able to establish a network connection with the robot's computer, the control logic will be executed on the robot.
+
+### Run Code Locally
+
+If you need to run [PID control loops](https://en.wikipedia.org/wiki/PID_controller) or other on-robot code, you can run control code on the same board that is running `viam-server`.
+
+To ensure intermittent internet connectivity does not interfere with the code's execution, there are some special steps you need to follow:
+
+{{< alert title="Note" color="note" >}}
+Currently, this only works with Python code which is running on the same board that `viam-server` is running on.
```

Comment:
I heard a rumor this might not even be required anymore btw - can we verify, and verify if it already works properly with other SDKs (like go - and by properly meaning it will automatically try to connect direct locally if internet is not available)

Reply:
@npentrel Tried to track down this ticket and could not find it, but I think you added this info in-- Do either of you know for sure, or have any advice as to who to reach out to to verify support here on Eng.?

Reply:
Unfortunately I don't know anything as to why the note is up

Reply:
I got clarification on this from Max - this is still required in this use-case, so ignore my comment.  Not sure about other language SDKs